apps/website/src/pages/docs/anchor.mdx

```diff
@@ -9,19 +9,37 @@ thumbnail: Anchor
 
 import PropsTable from '~/components/PropsTable.astro';
 import LiveExample from '~/components/LiveExample.astro';
-import Placeholder from '~/components/Placeholder.astro';
+import RelatedComponents from '~/components/RelatedComponents.astro';
 import * as AllExamples from '~/examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='typography-anchor' />
-
 <LiveExample src='Anchor.main.tsx' truncate={false}>
   <AllExamples.AnchorMainExample client:load />
 </LiveExample>
 
+The anchor's `href` attribute is used to indicate the link's destination while the text within the anchor element should describe the link's destination.
```

Comment:
distinction could be made more clear. "destination URL" vs "name of the link's name"